pallets/acurast/src/types.rs

```diff
@@ -78,21 +88,22 @@ pub enum ListUpdateOperation {
 
 /// Structure representing a job registration.
 #[derive(RuntimeDebug, Encode, Decode, TypeInfo, Clone, PartialEq)]
-pub struct JobRegistration<A, T>
+pub struct JobRegistration<AccountId, Extra, Reward>
 where
-    A: Parameter + Member + MaybeSerializeDeserialize + MaybeDisplay + Ord + MaxEncodedLen,
-    T: Parameter + Member + MaxEncodedLen,
+    AccountId: Parameter + Member + MaybeSerializeDeserialize + MaybeDisplay + Ord + MaxEncodedLen,
```

Comment:
Why are we actually requiring `Ord` trait?
If we need it, should we then add it to `#[derive()]`?

Reply:
I took the same constraint for `AccountId` in `frame_system::Config`